### PR TITLE
Copy the test data into the example dir before running tests

### PR DESCRIPTION
**Pull request by jsignell**
This is the other option mentioned in https://github.com/pyviz-topics/examples/pull/9#pullrequestreview-239505941

### CODE REVIEW

**Comment by jsignell** [comment]
I think this is the right approach. I'll extend it to add some data.

**Comment by jsignell** [comment]
I guess one remaining question is if this "small" data should be included in the github repo or put up on s3? Maybe s3 is better since most people won't want it, but it does make it slightly trickier to add an example.

File: dodo.py [hunk 1]
```diff
@@ -13,3 +15,73 @@ def task_ecosystem_setup():
...
+
+def task_small_data_setup():
+    """Copy small versions of the data from test_data"""
+
+    def copy_test_data(root='', name='attractors', test_data='test_data'):
+        print('Copying test data for {} ...'.format(name))
+        paths = _prepare_paths(root, name, test_data)
+
+        if not os.path.exists(paths['test']):
+            print("Fail: No test_data found for {} in {}".format(name, paths['test']))
```
Comment (jbednar) [inline dodo.py]: How will these failing conditions be detected by the tests, if they are just printing "Fail"?

Comment (jsignell) [reply dodo.py]: They won't. This was for debugging purposes. I should be raising proper errors.

**Comment by jsignell** [comment]
This is working how I want now. Just need to make a small version of the bay_trimesh data and then I'll merge

File: nyc_taxi/anaconda-project.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# After installing anaconda-project, this project can be reproduced using:
+#    anaconda-project run
+name: nyc_taxi
```
Comment (jbednar) [inline nyc_taxi/anaconda-project.yml]: Maybe the initial comment in these files could be simpler:

```
# To reproduce: install 'anaconda-project', then do 'anaconda-project run'.
```

Comment (jbednar) [reply nyc_taxi/anaconda-project.yml]: Or, if it has to fit into 80 characters exactly,

```
# To reproduce: install 'anaconda-project', then 'anaconda-project run'
```

**Comment by jsignell** [comment]
In the interest of migrating all the projects quickly I'm going to merge this and add the small data in a follow on PR.

**Pull request merged**